exporter/signalfxexporter/dpclient.go

```diff
@@ -189,10 +184,7 @@ func (s *sfxDPClient) retrieveAccessToken(md pdata.ResourceMetrics) string {
 	}
 
 	attrs := md.Resource().Attributes()
-	accessToken, ok := attrs.Get(splunk.SFxAccessTokenLabel)
-	if ok {
-		// Drop internally passed access token in all cases
-		attrs.Delete(splunk.SFxAccessTokenLabel)
```

Comment:
This is the necessary deletion I'm referring to.

Reply:
see previous comment.